### PR TITLE
feat(frontend): Display receipts on Block Details page

### LINKED ISSUE #590: Display receipts on the Block Details page
List the receipts that are included in a block (chunks) just like we list the transactions now.

This block has 1 transaction and 1 receipt (it is a local receipt, so it is "invisible" through the RPC): https://explorer.near.org/blocks/Ha8qhw396iJYfH9EUFBy2aoRNBR5TnqcZ1B1vPXPSmf2

This block (next one) has 0 transactions and 2 receipts (produced by the receipt in the previous block): https://explorer.near.org/blocks/2yAgnZ3CXW2iGFq3vppWtBe3PJSz5osLhtqbst7rNY7R

Implementation notes:
* We could use `block` and `chunk` JSON RPC endpoints to fetch the data from RPC, but we use Indexer database to fetch transactions, so I think it makes sense to use it for receipts as well
* Use `receipts`, `action_receipts`, `action_receipt_actions` tables to collect the data (add helpers in `src/libraries/explorer-wamp/receipts.ts`)

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Resolves:
- #590;
- #316;
- #390;
- #544   

### Test Plan:

- [x] All components are unit-tested
- [x] Update tests in existing componets
- [x] Manually check page loading

### CODE REVIEW

**Comment by shelegdmitriy** [comment]
There are list of receipts on blocks page. Also added a link to receipt on transaction (if such transaction is created). There is some things to do with Action* components so I'll carefully align it and cleaned up.

**Comment by shelegdmitriy** [comment]
@frol , does we need to show timestamp of receipt as it shown on TransactionActions?

File: frontend/src/components/utils/ExecutionReceiptStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { ReceiptExecutionStatus } from "../../libraries/explorer-wamp/receipts";
+
+const EXECUTION_RECEIPT_STATUSES: Record<ReceiptExecutionStatus, string> = {
+  Unknown: "Unknown",
+  Failure: "Failed",
+  SuccessValue: "Succeeded",
+  SuccessReceiptId: "SuccessReceiptId",
+};
```
Comment (shelegdmitriy) [inline frontend/src/components/utils/ExecutionReceiptStatus.tsx:8]: @frol, is this receipt statuses is valid?

File: frontend/src/components/utils/ExecutionReceiptStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { ReceiptExecutionStatus } from "../../libraries/explorer-wamp/receipts";
+
+const EXECUTION_RECEIPT_STATUSES: Record<ReceiptExecutionStatus, string> = {
+  Unknown: "Unknown",
+  Failure: "Failed",
+  SuccessValue: "Succeeded",
+  SuccessReceiptId: "SuccessReceiptId",
```
Comment (frol) [inline frontend/src/components/utils/ExecutionReceiptStatus.tsx:7]: Whenever we observe `SuccessReceiptId`, we should display something like "Status is deferred to receipt <receipt-id>"

File: frontend/src/components/utils/ReceiptHashLink.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import Link from "../utils/Link";
+
+export interface Props {
+  transactionHash?: string | null;
+  receiptId: string;
+  children?: React.ReactNode;
+}
+
+const ReceiptHashLink = ({ transactionHash, receiptId, children }: Props) => {
```
Comment (frol) [inline frontend/src/components/utils/ReceiptHashLink.tsx:9]:
```suggestion
const ReceiptLink = ({ transactionHash, receiptId, children }: Props) => {
```

**Review by frol — COMMENTED** [review]
I have not reviewed `explorer-wamp/receipts.ts` yet.

File: frontend/src/components/transactions/ActionRowBlock.tsx [hunk 4]
```diff
@@ -2,22 +2,23 @@ import React from "react";
...
+  actionLink?: React.ReactNode;
+  // actionBlock: any;
   viewMode: ViewMode;
   detalizationMode: DetalizationMode;
   className: string;
   icon: React.ReactElement;
   title: React.ReactElement | string;
   children?: React.ReactNode;
-  status?: T.ExecutionStatus;
+  status?: React.ReactNode;
```
[the remark this answers is not in the document]
Comment (shelegdmitriy) [reply frontend/src/components/transactions/ActionRowBlock.tsx:19]: To split statuses to different instances: Transaction and Receipt. That's why I pass `status` property as Component

**Comment by frol** [comment]
Bugs:

* [ ] the receipt link is inactive for transfer receipt: https://near-explorer-frontend-with-indexer-pr-qnuq.onrender.com/blocks/3pniBwf41PtBtjjesRp9RcFAWdqJbEA3txcPvaoS5oEf

**Comment by frol** [comment]
Bugs:

* [ ] the number of transactions and receipts do not match what I see on the page: https://near-explorer-frontend-with-indexer-pr-qnuq.onrender.com/blocks/3pniBwf41PtBtjjesRp9RcFAWdqJbEA3txcPvaoS5oEf (2 transactions and 4 receipts on the list reported as 8 transactions and 8 receipts in the top cells)

**Review by frol — APPROVED** [review]

**Pull request merged**